proposals/devcontainer-templates.md

```diff
@@ -0,0 +1,192 @@
+# Dev Container Templates reference
+
+Development container "Templates" are source files packaged together that encode configuration for a complete development environment. A Template can be used in a new or existing project and a [supporting tool](https://containers.dev/supporting) will use the configuration from the template to build a development container.
+
+Configuration is placed in a [`.devcontainer.json`](/docs/specs/devcontainer-reference.md#devcontainerjson). Alternatively, `.devcontainer/devcontainer.json` can also be used if the container needs to reference other files within the template, such as a `Dockerfile` or `docker-compose.yml`. A template can also provide additional source files (eg: boilerplate code or a [lifecycle script](https://containers.dev/implementors/json_reference/#lifecycle-scripts).
```

Comment:
Can a `.devcontainer.json` also reference other files? (I thought so, but the wording seems to suggest otherwise.)

Prepend "The" to "configuration"?

Reply:
Good catch, updated wordings.